### PR TITLE
docs: Link to Fider on CHEFS page

### PR DESCRIPTION
**Pull request by naomiaro**
# Description

Adding a link to Fider for CHEFS feature requests.

## Types of changes

Documentation (non-breaking change with enhancements to documentation)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have added necessary documentation (if appropriate)
- [x] I built and run locally to ensure no styles or formats break (if applicable)

## Further comments

As per the Gem file comment: `If you want to use GitHub Pages, remove the "gem "jekyll""`

Gem `github-pages` installs its own compatible version of jekyll so I've removed that separate gem install line from the README.

Screenshot 2021-12-14 at 10 30 29 AM

### CODE REVIEW

**Review by loneil — APPROVED** [review]
Changes look good assuming the gemfile stuff is all good.

File: Gemfile.lock [hunk 1]
```diff
@@ -239,28 +252,26 @@ GEM
...
-    unicode-display_width (1.7.0)
+    unf_ext (0.0.8)
+    unicode-display_width (1.8.0)
     wdm (0.1.1)
-    zeitwerk (2.4.0)
+    zeitwerk (2.5.1)
 
 PLATFORMS
-  ruby
-  x64-mingw32
```
Comment (jujaga) [inline Gemfile.lock]: Should we restore these lines? Not sure if they're autogenerated or not, but we do want this to be supported on multiple platforms. The darwin one is just for macOS.

**Review by jujaga — APPROVED** [review]

File: Gemfile.lock (hunk 1 above)
Comment (naomiaro) [reply Gemfile.lock]: hmm yeah just reading about that.

Comment (naomiaro) [reply Gemfile.lock]: https://www.moncefbelyamani.com/understanding-the-gemfile-lock-file/

**Pull request merged**